src/connection.toit

```diff
@@ -365,6 +365,12 @@ class DetachedSocket_ implements tcp.Socket:
   set_no_delay enabled/bool:
```

Comment:
I wonder if we can get rid of this now?

Reply:
I think we can remove it for the new version that requires a new SDK, ...